examples/ComplexExample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "RNExperimental",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.4.2",
+    "react-native": "0.40.0",
+    "react-native-vector-icons": "^4.0.0",
+    "react-navigation": "git+https://github.com/react-community/react-navigation.git",
```

Comment:
@si-harps This may be relevant to your interests https://github.com/react-community/react-navigation/pull/900#issuecomment-291360232